newsroom/templates/base_layout.html

```diff
@@ -61,10 +61,10 @@ <h3 class="a11y-only">{{ gettext('Side Navigation') }}</h3>
                         {% for nav in sidenavs(request.blueprint)|selectattr("group", "equalto", group) %}
                         {% if group > 0 and loop.first and loop.length %}
                         <li class="sidenav-icons__separator">
-                            <span className="separator__dot" aria-hidden="true"></span>
-                            <span className="separator__dot" aria-hidden="true"></span>
-                            <span className="separator__dot" aria-hidden="true"></span>
-                            <span className="separator__dot" aria-hidden="true"></span>
+                            <span class="separator__dot" aria-hidden="true"></span>
```

Comment:
Is there a reason to use 3 separate `spans` for these dots, as opposed to using `<i class="icon--more icon--rotate-90 icon--gray"></i>`?